### PR TITLE
lib: modem_key_mgmt: Handle nrf_sec_tag_t as unsigned

### PR DESCRIPTION
**Pull request by stig-bjorlykke**
Use "%u" as format for unsigned nrf_sec_tag_t.

Signed-off-by: Stig Bjørlykke <stig.bjorlykke@nordicsemi.no>

### CODE REVIEW

File: lib/modem_key_mgmt/modem_key_mgmt.c [hunk 1]
```diff
@@ -90,7 +90,7 @@ static int key_fetch(nrf_sec_tag_t tag,
 	cmee_enable(&cmee_was_active);
 
 	err = nrf_modem_at_cmd(scratch_buf, sizeof(scratch_buf),
-			       "AT%%CMNG=2,%d,%d", tag, cred_type);
+			       "AT%%CMNG=2,%u,%u", tag, cred_type);
```
Comment (rlubos) [inline lib/modem_key_mgmt/modem_key_mgmt.c:93]: `nrf_sec_tag_t` is unsigned, so using `%u` is understandable, but shouldn't the format specifier for `modem_key_mgmt_cred_type` be left as is? It's `enum` so it is signed int.

Comment (lemrey) [reply lib/modem_key_mgmt/modem_key_mgmt.c:93]: I agree with that

Comment (stig-bjorlykke) [reply lib/modem_key_mgmt/modem_key_mgmt.c]: Fixed

**Review by rlubos — APPROVED** [review]

**Comment by rlubos** [comment]
I wonder if we need a rebase here, there are build failures in the `nrf-iot_libmodem` plan:
> [2022-09-01T08:24:16.496Z] /tmp/tmpfadj71c1.conf:2: warning: attempt to assign the value 'y' to the undefined symbol NRF_MODEM_LIB_TRACE

**Pull request merged**